modules/tinymce/src/themes/silver/test/ts/headless/components/imagepreview/BasicImagePreviewTest.ts

```diff
@@ -0,0 +1,118 @@
+import { ApproxStructure, Assertions, UiFinder } from '@ephox/agar';
+import { AlloyComponent, GuiFactory, Representing, TestHelpers } from '@ephox/alloy';
+import { describe, it } from '@ephox/bedrock-client';
+import { Optional } from '@ephox/katamari';
+import { Ready } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import { ImagePreviewData, renderImagePreview } from 'tinymce/themes/silver/ui/dialog/ImagePreview';
+
+describe('headless.tinymce.themes.silver.components.imagepreview.BasicImagePreviewTest', () => {
+  const testImageUrl = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8DwHwAFBQIAX8jx0gAAAABJRU5ErkJggg==';
+
+  const hook = TestHelpers.GuiSetup.bddSetup((_store, _doc, _body) => GuiFactory.build(
+    {
+      dom: {
+        tag: 'div',
+        styles: {
+          width: '200px'
+        }
+      },
+      components: [
+        renderImagePreview({
+          name: 'test-panel',
+          height: Optional.some('200px'),
+        })
+      ]
+    }
+  ));
+
+  const findImage = (component) => UiFinder.findIn<HTMLImageElement>(component.element, 'img').getOrDie();
+
+  const assertImageState = (label: string, component: AlloyComponent, url: string) => {
+    const node = findImage(component).dom;
+    assert.equal(node.src, url, label + ' - checking "src" attribute');
+  };
+
+  const setPanelState = (component: AlloyComponent, state: ImagePreviewData) => {
+    // the component is a wrapper div just there to set a known width, the image preview is the child
+    const [ imagePreview ] = component.components();
+    Representing.setValue(imagePreview, state);
+  };
+
+  it('Check basic structure', () => {
```

Comment:
Should these get Jira numbers in the test names? And are you able to phrase these as sentences? Like
```suggestion
  it('TINY-1234: checks basic structure', () => {
```

Reply:
We had a bit of a conversation about this earlier in the PR, and it's because I used an existing test as the template:
https://github.com/tinymce/tinymce/blob/0325f25d77df58040e6da4af890999b6f20a963a/modules/tinymce/src/themes/silver/test/ts/headless/components/checkbox/BasicCheckboxTest.ts#L44

So we probably need to come up and apply proper standards to all of the component tests at some point.